_data/cloudservices.yml

```diff
@@ -3442,27 +3439,24 @@ services:
             ref: https://cloud.google.com/dialogflow-enterprise/
             icon: Natural-Language-API.png
       - ibm:
-          - name: Natural Language Classifier
-            ref: https://cloud.ibm.com/docs/services/natural-language-classifier?topic=natural-language-classifier-natural-language-classifier
+          - name: Natural Language Understanding
+            ref: https://cloud.ibm.com/catalog/services/natural-language-understanding
             icon: ibmcloud.png
           - name: Language Translator
-            ref: https://cloud.ibm.com/docs/services/language-translator?topic=language-translator-gettingstarted
```

Comment:
Delete this. NeuralSeek is a 3rd party
> If you are using IBM Watson Language Translator in OpenPages® version 9.0.0.2 or later, consider switching to NeuralSeek as your language translation service because the IBM Watson Language Translator service has been deprecated.